indico/modules/rb_new/client/styles/util.scss

```diff
@@ -0,0 +1,42 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+@import 'base/palette';
+
+$white: #fff;
+$highlight-color: #7dcfb6;
+$primary-color: #2185d0;
+$indico-top-bar-height: 50px;
+$menu-height: 75px;
+$rb-content-padding: 2em;
+$sui-font-family: Lato, 'Helvetica Neue', Arial, Helvetica, sans-serif;
+$pre-booking-stripe-colors: (#f2711c, #f48d49);
+$pre-booking-conflict-stripe-colors: (#db2828, #e56868);
```

Comment:
Any reason why you didn't choose to use `lighten(...)` and `darken(...)`? That would make things more readable.

Reply:
Yup, those are based on the colors SUI is using